src/organizations/components/OrgProfileTab.tsx

```diff
@@ -43,105 +49,175 @@ const OrgProfileTab: FC = () => {
-          <h4>Common IDs</h4>
+          <h4>Organization Profile</h4>
 
           <Heading
             element={HeadingElement.H4}
-            style={{marginBottom: '4px'}}
             weight={FontWeight.Regular}
+            style={headingStyle}
           >
-            User ID
+            Name
           </Heading>
-          <div className="code-snippet" data-testid="code-snippet--userid">
-            <div className="code-snippet--text">
-              <pre>
-                <code>{me.id}</code>
-              </pre>
-            </div>
-            <FlexBox
-              className="code-snippet--footer"
-              margin={ComponentSize.Medium}
-            >
-              <CopyButton
-                text={me.id}
-                testID="copy-btn--userid"
-                onCopy={generateCopyText('User ID', me.id)}
-              />
-              <label className="code-snippet--label">
-                {`${me.name} |`} <b>User ID</b>
-              </label>
-            </FlexBox>
-          </div>
+          <FlexBox direction={FlexDirection.Row} margin={ComponentSize.Medium}>
+            <Input
+              value={org.name}
+              style={{width: 'max-content'}}
+              data-testid="danger-zone--org-name"
+              testID="danger-zone--org-name"
+            ></Input>
+            <Button
+              testID="rename-org--button"
+              text="Rename"
+              icon={IconFont.Pencil}
+              onClick={handleShowEditOverlay}
+            />
+          </FlexBox>
+          <FlexBox
+            direction={FlexDirection.Row}
+            margin={ComponentSize.Medium}
+            justifyContent={JustifyContent.SpaceBetween}
+            stretchToFitWidth={true}
+            style={{width: '85%'}}
+          >
+            {[
+              {
+                label: 'Provider',
+                src: convertProviderSymbol(me.quartzMe?.billingProvider),
+              },
+              {label: 'Region', src: me.quartzMe?.regionCode},
+              {label: 'Location', src: me.quartzMe?.regionName},
+            ].map(({label, src}) => {
+              if (!!src) {
+                return (
+                  <FlexBox
+                    key={`org-${label.toLowerCase()}`}
+                    direction={FlexDirection.Column}
+                    margin={ComponentSize.Large}
+                    alignItems={AlignItems.FlexStart}
+                  >
+                    <Heading
+                      element={HeadingElement.H4}
+                      weight={FontWeight.Regular}
+                      style={headingStyle}
+                    >
+                      {label}
+                    </Heading>
+                    <span style={{fontWeight: FontWeight.Light, width: '100%'}}>
+                      {src}
+                    </span>
+                  </FlexBox>
+                )
+              } else {
+                return null
+              }
+            })}
```

Comment:
nit: can we make this its own self-container piece of logic. It seems like mapping over the data and the flex component can be contained in their own component that handles all this and compartmentalizes the logic

Reply:
done. Broken out OrgProfileTab into own dir, and made various child/reusable components.